app/actions/configurations.py

```diff
@@ -8,7 +8,7 @@ class AuthenticateConfig(AuthActionConfiguration):
 
 
 class PullEventsConfig(PullActionConfiguration):
-    carto_url: str = "https://rw-nrt.carto.com/api/v2/sql"
+
     gfw_share_link_url: pydantic.HttpUrl = pydantic.Field(
```

Comment:
Can this be a breaking change? I'm wondering how this will affect pre-existent connections in prod as they will have a config with `carto_url` field instead of `gfw_share_link_url`. We could write some data migration as needed.

Reply:
It's OK. That property is no longer used.